articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+
+<!--more-->
+
+However, for more complex multi-page websites, a static site may not be suitable. Every time you want to add content, a new page has to be created and coded. If you change something that appears on multiple pages then you’d have to change it everywhere. That’s where static site generators come in. They bring the power of templating to static sites.
+
+There are many static site generators, [Hugo](https://gohugo.io), [Gatsby](http://gatsbyjs.org) and [Next.js](http://gatsbyjs.org) to name a few but we will use Eleventy because it’s easy to use and flexible. It uses vanilla JavaScript and allows multiple template languages to be mixed and matched so you don’t have to learn a new language or templating system. Static site generators work by taking the layout templates you’ve created and building the pages using the template and data you’ve provided.
+
+**Swap these paragraphs around?**
+
+Templating (see Converting a Static Site to a Dynamic NodeJS Web App **link** for more info **maybe link later when I mention EJS**) lets you reuse code by inserting it into multiple places and add dynamic data. There are a variety of templating languages such as [Nunchucks](https://mozilla.github.io/nunjucks), [Liquid](https://shopify.github.io/liquid), [Pug](https://pugjs.org/api/getting-started.html) and [Handlebars](https://handlebarsjs.com) but the one we’ll be using is [EJS](https://ejs.co).
+
+### Installing Eleventy
+
+The first step is to install Eleventy which will be done through the package manager, NPM. New to NPM? Check out my installation guide **(link to NPM install in Converting a Static Site to a Dynamic NodeJS Web App article)**.
+
+Create a package.json file using `npm init -y` and then install Eleventy using `npm install --save-dev @11ty/eleventy`.
+
+To test the install, run `npx @11ty/eleventy` which should return:
+`Processed X files in number of seconds (version number)`.
+
+### Serving Your Static Site using Eleventy
+
+The second step is to be able to serve your current site using Eleventy.
+
+To do so, you'll have to create a config file and add some settings to it which will tell Eleventy how to handle your images, CSS and JavaScript.
+
+In the root folder of your site, create a JS file called .eleventy.js and include the following:
+
+```js
+module.exports = function (eleventyConfig) {
+    eleventyConfig.addPassthroughCopy('img');
+    eleventyConfig.addPassthroughCopy('css');
+    eleventyConfig.addPassthroughCopy('js');
+    return {
+        passthroughFileCopy: true
+    };
+};
+```
+This will tell Eleventy when it builds your website to just copy the img, CSS and js folders across (which is called passthrough copy) so you can link them to them in your code.
```

Comment:
"...so you can link them to them in your code." Revision needed. Nit-picky, but also consider getting rid fo the "just" in this sentence.

Reply:
Corrected and adjusted wording as suggested.